magento/pwa-studio
# [feature]: Display "no payment method available" message if no allowed or configured payment methods are present

## Description

It's not showing message when there are no supported payment methods available.
It's becoming and empty container instead of showing proper message.

## Related Issue





Closes #2854.
[PWA-1115](https://jira.corp.magento.com/browse/PWA-1115)
## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. Make sure you have braintree payment method disabled and other payment methods like Purchase order, bank transfer enabled.
2. Add any product to cart
3. Go to Cart -> Checkout
4. Add shipping address and select any shipping method
5. Let it load the payment methods from server. After loading completion payment method it should show the message.

## Screenshots / Screen Captures (if appropriate)
[![Image from Gyazo](https://i.gyazo.com/ff4cba4f3bb6f739c187eb7edc47d451.png)](https://gyazo.com/ff4cba4f3bb6f739c187eb7edc47d451)

[![Image from Gyazo](https://i.gyazo.com/a8b9de00d88079b2c38fbc6a1e018fc3.png)](https://gyazo.com/a8b9de00d88079b2c38fbc6a1e018fc3)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
